omero/sysadmins/limitations.rst

```diff
@@ -43,15 +43,28 @@ user, you cannot make the group into a private group.
 File format support
 -------------------
 
+Large images
+^^^^^^^^^^^^
+
+When you import an image over a certain size, OMERO will generate a Pyramid of lower resolution
```

Comment:
maybe we worth saying that its only when the XY dimensions exceed a certain size

Reply:
That's covered by the next sentence.